getting-started/coaddition.rst

```diff
@@ -156,15 +156,19 @@ Putting this together, run the following command to warp ``HSC-R``-band exposure
    makeCoaddTempExp.py DATA --rerun coadd \
        --selectId filter=HSC-R \
        --id filter=HSC-R tract=0 patch=0,0^0,1^0,2^1,0^1,1^1,2^2,0^2,1^2,2 \
-       --config doApplyUberCal=False doApplySkyCorr=False
+       --config doApplyExternalPhotoCalib=False doApplyExternalSkyWcs=False \
+       doApplySkyCorr=False
 
 .. tip::
 
    :command:`makeCoaddTempExp.py` automatically filters out exposures that don't fit on a patch.
 
 .. note::
 
-   Since this tutorial doesn't prepare an uber calibration or sky correction, you needed to explicitly disable these calibration steps from the default HSC processing configuration.
+   Since this tutorial doesn't prepare an uber calibration (producing
```

Comment:
Could you unwrap the lines in this paragraph? The text needs to be one-sentence-per-line, just like the rest of the document.

Reply:
Additionally, since the name has now changed to External instead of Uber, can the text here please say "Since this tutorial doesn't prepare an external calibration" (or equivalent)